src/SortableListAdmin/AbstractSortableListAdmin.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/*
+ *
+ * This file is part of the Sonata for Ekino project.
+ *
+ * (c) 2018 - Ekino
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ */
+
+namespace Sonata\SonataHelpersBundle\SortableListAdmin;
+
+use Sonata\AdminBundle\Admin\AbstractAdmin;
+use Sonata\AdminBundle\Route\RouteCollection;
+use Symfony\Bridge\Doctrine\RegistryInterface;
+
+/**
+ * Class AbstractSortableListAdmin.
+ *
+ * @author  Benoit de Jacobet <benoit.de-jacobet@ekino.com>
+ */
+abstract class AbstractSortableListAdmin extends AbstractAdmin
+{
+    // Sortable list can be only with one page
+    protected $maxPerPage = 500;
+    protected $perPageOptions = [500];
+
+    protected $datagridValues = [
+        '_page' => 1,
+        '_sort_order' => 'ASC',
+        '_sort_by' => 'position',
+    ];
+
+    /**
+     * @var RegistryInterface
+     */
+    protected $doctrine;
+
+    /**
+     * Set Registry.
+     *
+     * @param RegistryInterface $doctrine
+     */
+    public function setRegistry(RegistryInterface $doctrine)
+    {
+        $this->doctrine = $doctrine;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getTemplate($name)
+    {
+        switch ($name) {
+            case 'list':
+                return 'CanalPlusAwakenAdminBundle:CRUD:list_sortable.html.twig';
```

Comment:
should be `AppBundle` and updated by user when he install it. It must be indicated in documentation.

Reply:
Remove reference to customer please